src/components/MutationName.tsx

```diff
@@ -0,0 +1,186 @@
+    text: 'RNA-dependent RNA polymerase, forms complex with Nsp8 (primase)',
+    name: 'Nsp7',
+  },
+  {
+    range: [3943, 4140],
+    text: 'RNA polymerase, replicase, primase',
+    name: 'Nsp8',
+  },
+  {
+    range: [4141, 4253],
+    text: 'single-stranded RNA-binding protein',
+    name: 'Nsp9',
+  },
+  {
+    range: [4254, 4392],
+    text: 'growth-factor-like protein containing Zinc-binding motifs, important for mRNAs cap methylation',
+    name: 'Nsp10',
+  },
+  {
+    range: [4393, 5324],
+    text:
+      'RNA dependent RNA polymerase (catalytic subunit), responsible for replication and transcription of viral RNA',
+    name: 'Nsp12',
+  },
+  {
+    range: [5325, 5925],
+    text: 'Zinc-binding domain, NTPase/helicase',
+    name: 'Nsp13',
+  },
+  {
+    range: [5926, 6452],
+    text: "Proofreading exoribonuclease domain (3' to 5' direction)",
+    name: 'Nsp14',
+  },
+  {
+    range: [6453, 6798],
+    text: 'EndoRNAse, Mn(2+) dependent endoribonuclease',
+    name: 'Nsp15',
+  },
+  {
+    range: [6799, 7096],
+    text: "2'-O-ribose methyltransferase",
+    name: 'Nsp16',
+  },
+];
+
+const Mut = styled.span`
+  &:hover {
+    background: #eaeaea;
+    border-radius: 15px;
+    padding: 5px;
+    cursor: pointer;
+  }
+`;
+
+const getInformation = (gene: string, position: number): { header: string; text: JSX.Element } => {
+  let header: string | undefined = headerMap[gene];
+  if (!header) {
+    return {
+      header: gene + ' protein',
+      text: (
+        <p>
+          <i className='text-muted'>No description available</i>
+        </p>
+      ),
+    };
+  }
+
+  let text;
+  if (gene === 'ORF1a' || gene === 'ORF1b') {
+    if (gene === 'ORF1b') {
+      position += 4401;
+    }
+    for (let nsp of ORF1abProteins) {
+      if (position >= nsp.range[0] && position <= nsp.range[1]) {
+        header += ': ' + nsp.name;
+        text = <p>{nsp.text}</p>;
+      }
+    }
+    if (!text) {
+      text = (
+        <p>
+          <i className='text-muted'>No description available</i>
+        </p>
+      );
+    }
+  } else {
+    text = <p>{textMap[gene]}</p>;
+  }
+
+  return { header, text };
+};
+
+export const MutationName = ({ mutation }: Props) => {
+  const [gene, mut] = mutation.split(':');
+  const position = mut.substring(1, mut.length - 1);
+
+  const information = getInformation(gene, parseInt(position));
+  const popover = (
+    <Popover id='popover-basic'>
+      <Popover.Title as='h3'>{information.header}</Popover.Title>
+      <Popover.Content>
+        <div style={{ width: '600px' }}>{information.text}</div>
+      </Popover.Content>
+    </Popover>
+  );
+
+  return (
+    <>
+      {/*TODO It does not seem right to overwrite the CSS here. But how else?*/}
+      <style type='text/css'>
+        {`
+          .popover {
+            max-width: none;
+          } 
+        `}
+      </style>
+      <OverlayTrigger trigger='click' overlay={popover} rootClose={true} transition={false}>
```

Comment:
Some of the larger popups get slightly cut off. Adding `placement='bottom'` should be good enough, since we have plenty of space below.

![cut-off-popup](https://user-images.githubusercontent.com/1489115/110167418-038e0d00-7df6-11eb-9f03-5782420d369b.png)